<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>569</width>
    <height>364</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralWidget">
   <layout class="QVBoxLayout" name="verticalLayout">
    <item>
     <widget class="QPlainTextEdit" name="plainTextEdit">
      <property name="plainText">
       <string/>
      </property>
     </widget>
    </item>
    <item>
     <widget class="QWidget" name="widget" native="true">
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <spacer name="horizontalSpacer">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>126</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
       <item>
        <widget class="QPushButton" name="btnPortOC">
         <property name="text">
          <string>Open port</string>
         </property>
        </widget>
       </item>
       <item>
        <widget class="QPushButton" name="btnClear">
         <property name="text">
          <string>Clear</string>
         </property>
        </widget>
       </item>
      </layout>
     </widget>
    </item>
   </layout>
  </widget>
  <widget class="QMenuBar" name="mnuMenu">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>569</width>
     <height>18</height>
    </rect>
   </property>
   <widget class="QMenu" name="menu1">
    <property name="title">
     <string>Menu</string>
    </property>
    <widget class="QMenu" name="menuPorts">
     <property name="title">
      <string>Ports</string>
     </property>
    </widget>
    <addaction name="actionLoad"/>
    <addaction name="menuPorts"/>
   </widget>
   <widget class="QMenu" name="menuSend">
    <property name="enabled">
     <bool>true</bool>
    </property>
    <property name="title">
     <string>Send</string>
    </property>
    <addaction name="actionSETUP"/>
    <addaction name="actionPOWER_OFF"/>
    <addaction name="actionSTOP"/>
    <addaction name="actionSTART"/>
    <addaction name="actionSENSOR_INIT"/>
    <addaction name="actionREFERENCE"/>
    <addaction name="actionMOTOR_SETUP"/>
   </widget>
   <addaction name="menu1"/>
   <addaction name="menuSend"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionLoad">
   <property name="text">
    <string>Load</string>
   </property>
  </action>
  <action name="actionSETUP">
   <property name="text">
    <string>SETUP</string>
   </property>
  </action>
  <action name="actionPOWER_OFF">
   <property name="text">
    <string>POWER OFF</string>
   </property>
  </action>
  <action name="actionSTOP">
   <property name="text">
    <string>STOP</string>
   </property>
  </action>
  <action name="actionSTART">
   <property name="text">
    <string>START</string>
   </property>
  </action>
  <action name="actionSENSOR_INIT">
   <property name="text">
    <string>SENSOR INIT</string>
   </property>
  </action>
  <action name="actionREFERENCE">
   <property name="text">
    <string>REFERENCE</string>
   </property>
  </action>
  <action name="actionMOTOR_SETUP">
   <property name="text">
    <string>MOTOR SETUP</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources/>
 <connections/>
</ui>
